src/mobx/stores/VaultStore.ts

```diff
@@ -176,104 +209,8 @@ export default class VaultStore {
 			return vault ? [vault] : [];
 		});
 
-		if (this.vaultsFilters.hidePortfolioDust) {
```

Comment:
this is just moving the code to a private method. it was getting too long and the flags made it unreadable